.github/workflows/user-service-deploy.yml

```diff
@@ -56,6 +57,7 @@ jobs:
             sudo docker container ls -la --format "{{.Names}}" | grep user-service | xargs --no-run-if-empty sudo docker stop user-service
             sudo docker container ls -la --format "{{.Names}}" | grep user-service | xargs --no-run-if-empty sudo docker rm user-service
             sudo docker run \
+              --network ${{ env.DOCKER_NETWORK_BRIDGE_NAME }} \
```

Comment:
Added the `user-service` docker image to the network so that the `webserver` can access the API.